args.go

```diff
@@ -146,6 +146,7 @@ func getAWSLogsArgs() (*awslogs.Args, error) {
 		Stream:              stream,
 		CredentialsEndpoint: credentialsEndpoint,
 		CreateGroup:         viper.GetString(awslogs.CreateGroupKey),
+		SkipCreateStream:    viper.GetString(awslogs.SkipCreateStreamKey),
```

Comment:
Did you consider calling this `CreateStream`, and having the default value as "true" in Moby? I think it would flow better that way and be easier to understand. I don't have strong opinions, however.

Reply:
Yeah `CreateStream` sounds good to me as well. I don't have preference and if people feel `CreateStream` is better, I will update accordingly.

Reply:
Yes I agree that `CreateStream` defaulting to true is a more intuitive name.

Reply:
sounds good, I will update the moby fork and update the pr